jscomp/ml/ast_uncurried.ml

```diff
@@ -69,14 +69,20 @@ let coreTypeIsUncurriedFun (typ : Parsetree.core_type) =
     true
   | _ -> false
 
-let typeIsUncurriedFun = Ast_uncurried_utils.typeIsUncurriedFun
-
-let typeExtractUncurriedFun (typ : Parsetree.core_type) =
+let coreTypeExtractUncurriedFun (typ : Parsetree.core_type) =
```

Comment:
Renamed from typeExtractUncurriedFun coreTypeExtractUncurriedFun to follow naming convention in this file. I I have redefined  typeExtractUncurriedFun below. (coreTypeIsUncurriedFun is named above fore the same paramater type, and for functions with Types.type_expr as a param use typeIsUncurredFun and typeExtractUncurriedFun respectively)